### ISSUE TITLE
Licensing documentation

### ISSUE THREAD

**Issue by theseyi** [seanherron/Flag-Webicons#10]
Could you create a license file in the root folder so it's clear to users how they're able to use this repo and what their liabilities are? I'm assuming it's creative commons? Cheers

**Comment by sDaniel** [comment]
Yes please otherwise the project can't be used in most projects.

**Comment by theseyi** [comment]
@seanherron
